### DISCUSSION TITLE
建议可以出一个指导性的wiki

### DISCUSSION THREAD

**Discussion by ColdeZhang** [WeNeedHome/SummaryOfLoanSuspension#118] [Ideas]
可以添加一个指导性的wiki，里面包含一些撰写告知书需要注意的事项、需要收集的信息、具体的流程等等，便于其他楼盘的快速跟进。目前看来很多楼盘的告知书其实写的并不严谨，队伍还不够有序。严谨有序的队伍才能打胜仗！

**Comment by Mmmmeruem** [comment]


**Reply by ColdeZhang** [reply]
Non-cooperation movement

**Comment by gu-fan** [comment]
指导。。。有统计就不错了，你还想指导。。。

**Comment by ppinkiris** [comment]
这种帖子才是大家需要的 支持
有没有断供的业主出来share一下的
